moveit_ros/planning/planning_scene_monitor/src/planning_scene_monitor.cpp

```diff
@@ -606,11 +606,11 @@ void PlanningSceneMonitor::updatePublishSettings(bool publish_geom_updates, bool
 {
   PlanningSceneMonitor::SceneUpdateType event = PlanningSceneMonitor::UPDATE_NONE;
   if (publish_geom_updates)
-    event = (PlanningSceneMonitor::SceneUpdateType)((int)event | (int)PlanningSceneMonitor::UPDATE_GEOMETRY);
+    event = (PlanningSceneMonitor::SceneUpdateType)(static_cast<int>(eve)nt | static_cast<int>(Pla)nningSceneMonitor::UPDATE_GEOMETRY);
   if (publish_state_updates)
-    event = (PlanningSceneMonitor::SceneUpdateType)((int)event | (int)PlanningSceneMonitor::UPDATE_STATE);
+    event = (PlanningSceneMonitor::SceneUpdateType)(static_cast<int>(eve)nt | static_cast<int>(Pla)nningSceneMonitor::UPDATE_STATE);
   if (publish_transform_updates)
-    event = (PlanningSceneMonitor::SceneUpdateType)((int)event | (int)PlanningSceneMonitor::UPDATE_TRANSFORMS);
+    event = (PlanningSceneMonitor::SceneUpdateType)(static_cast<int>(eve)nt | static_cast<int>(Pla)nningSceneMonitor::UPDATE_TRANSFORMS);
```

Comment:
Similar issues with the parenthesis in the wrong place like the other ones I pointed out above.

Reply:
I resolved this issue. Should I make a new PR for it ?  >